### PR TITLE
miniget: Add version 1.1.10.1830

### PR DESCRIPTION
**Pull request by wenmin92**
[MiniGet](http://www.miniget001.com/): a Lightweight and Fast Download Manager & BitTorrent Client

### CODE REVIEW

File: bucket/miniget.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "http://www.miniget001.com/",
+    "description": "a Lightweight and Fast Download Manager & BitTorrent Client.",
```
Comment (Ash258) [inline bucket/miniget.json]:
```suggestion
    "description": "A lightweight and fast download manager & BitTorrent client.",
```

**Comment by r15ch13** [comment]
Website is offline.

**Comment by Ash258** [comment]
No. It is available 

A

**Comment by r15ch13** [comment]
Strange, I got 404 when tested back then. :sweat_smile:

**Comment by wenmin92** [comment]
> No persist for json file and config, Torrent folders?

Thanks, I will check later.

**Comment by Ash258** [comment]
Is there any specific dependency? On my workstation at work I am experiencing this:


image

**Comment by wenmin92** [comment]
> Is there any specific dependency? On my workstation at work I am experiencing this:

I have asked the author about this problem, but not replied yet. I think this may be a package issue, the exe installer is Ok. We can wait some days or use the exe version(can be extract by 7z).

**Comment by issaclin32** [comment]
> Is there any specific dependency? On my workstation at work I am experiencing this:

I got the same problem. When I search in the menu and set the language to *English*, the problem was solved.

I guess this problem will happen to all the PCs with a **default language that is not supported by Miniget**. `miniget` did not handle these situation properly (which is to set the language to *English*),  therefore all the dialogs appears to be empty. (I have not test it on a PC with English interface, so I can not be completely sure.)

It is possible to fix to problem by manually tweaking `MiniGetConfig.cfg`, but that will be error-prone and hard to maintain.
hex

**Comment by wenmin92** [comment]
Hi @issaclin32 , Thanks for your test. My computer Language is English while Region is China. Still not work.

**Comment by wenmin92** [comment]
Sorry for lost update. I'm guessing whether the author might update, so I went to the website today and found that there was indeed a small update on December 28. Tested, this new updated version can be used normally.

**Comment by Ash258** [comment]
I will test again after I am back in the office on all workstations.

**Comment by r15ch13** [comment]
Off-Topic: Regarding my network problem with the domain.
It gets blocked by the [maleware list](https://hosts-file.net/default.asp?s=www.miniget001.com) from [hpHosts](https://hosts-file.net/). It was added in 02-04-2018.

**Comment by Ash258** [comment]
Default language after clean install is Espanol.

image

@wenmin92 Any response from author about https://github.com/lukesampson/scoop-extras/pull/3319#issuecomment-575765996 ?

**Comment by wenmin92** [comment]
There is no other response. 
My clean install language is Chinese. I think the language is related to the system language setting.

**Comment by rashil2000** [comment]
@issaclin32 @wenmin92 any updates on this?

**Comment by issaclin32** [comment]
The official website (http://www.miniget001.com/) has been discontinued.
Also I cannot find a new one.

Normally I will try to find a file that matches the same hash, and add it to [Binary](https://github.com/ScoopInstaller/Binary).

But for the case of **MiniGet**, I would not do that because the software involves outbound connections, and it is risky to use an outdated one.

**Comment by rashil2000** [comment]
Oh, thanks for the quick update. Closing this then.

**Pull request closed without merging**